### DISCUSSION TITLE
No phone number

### DISCUSSION THREAD

**Discussion by svenroblox** [vercel/community#784] [Help] [answered]
Hello, I don't have a phone number, so I can't access vercel?

**Comment by amyegan** [comment, marked as answer]
Since account registration requires personal information, we should handle this in a ticket instead of this public discussion. Please contact Vercel Support by emailing [registration@vercel.com](mailto:registration@vercel.com) and include details such as the email address or username you're trying to use and a description of the project you are looking to deploy with Vercel.
